### PR TITLE
Add StorPool to the list of drivers

### PR DESCRIPTION
**Pull request by sp-nikolay-tenev**

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [SPDK-CSI](https://github.com/spdk/spdk-csi) | `csi.spdk.io` | v1.1 | A Container Storage Interface (CSI) Driver for [SPDK](https://spdk.io/) | Persistent and Ephemeral | Read/Write Single Pod | Yes |
 [StorageOS](https://docs.storageos.com/docs/platforms/kubernetes/install/) | `storageos` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for [StorageOS](https://storageos.com/) | Persistent | Read/Write Multiple Pods | Yes |
 [Storidge](https://docs.storidge.com/kubernetes_storage/overview.html) | `csi.cio.storidge.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for [Storidge CIO](https://storidge.com/) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion
+[StorPool](https://storpool.com/kubernetes) | `csi-driver.storpool.com` | v1.0 | A Container Storage Interface (CSI) Driver for [StorPool](https://storpool.com/) | Persistent and Ephemeral | Read/Write Multiple Pods | Yes | Expansion
```
Comment (xing-yang) [inline book/src/drivers.md:91]: Looks like "https://storpool.com/kubernetes" only has a very high level overview.  Do you have a website that shows how to use your CSI driver?

Comment (sp-nikolay-tenev) [reply book/src/drivers.md:91]: Yes, it is here - https://kb.storpool.com/storpool_integrations/github/kubernetes.html

Comment (xing-yang) [reply book/src/drivers.md:91]: Great.  Can you use this link instead of "https://storpool.com/kubernetes"?

Comment (sp-nikolay-tenev) [reply book/src/drivers.md]: Done.

**Pull request merged**